ouroboros-consensus/src/ouroboros-consensus/Ouroboros/Consensus/Storage/LedgerDB/Config.hs

```diff
@@ -74,15 +75,35 @@ data SnapshotInterval =
   | RequestedSnapshotInterval DiffTime
   deriving stock (Eq, Generic, Show)
 
--- | The number of keys to read /at most/ in a backing store range query.
+-- | The number of diffs in the immutable part of the chain that we have to see
+-- before we flush ledger state to disk. See 'onDiskShouldFlush'. It can be:
 --
--- INVARIANT: Should be at least 1!
+-- 1. either explicitly provided by a user in the number of diffs in the
+--    immutable part of the chain.
+-- 2. a default value, which is determined by a specific 'DiskPolicy'. See
+-- 'defaultDiskPolicy' as an example.
+--
+-- INVARIANT: Should be at least 0.
+data FlushFrequency =
+    DefaultFlushFrequency
+  | RequestedFlushFrequency Word64
```

Comment:
Would it make sense to repeat that the `Word64` is the maximum number of diffs in the immutable part before we flush?

Reply:
Done